.eslintrc.js

```diff
@@ -95,6 +77,12 @@ module.exports = {
     'no-self-compare': 0,
     'no-underscore-dangle': 0,
     'prefer-destructuring': 0,
+    
+    /* tsx */
+    "@typescript-eslint/no-var-requires": 1,
+    "@typescript-eslint/no-empty-function": 1,
+    "@typescript-eslint/ban-types": 1,
+    "@typescript-eslint/ban-ts-comment": 1,
```

Comment:
please explain the purpose of these settings. Are they different from some defaults?

I am all for enabling more rues, but it seems this section was previously used to _disabled_ some of them, so I am not clear why we need to enable others, and why specifically these 4 and not more.

Reply:
Because, the new lint is giving error of these 4 types. So, I added these and convert the error to warning. Eslint by default doesn't handle these settings, it's always giving error if we do not handle it.